shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1113,10 +1096,6 @@ export default {
           entry.config = await entry.config.save();
         }
 
-        if ( !entry.pool.hostnamePrefix ) {
```

Comment:
Why did this go? Not needed?

Reply:
No, from what I can tell that field doesn't exist: the API response to the POST request doesn't include it nor is it listed in the relevant schema. I can see it referenced in rke1 provisioning, so I'm guessing that there was just a false assumption it'd be needed in machine pools too.